common/flatpak-dir.c

```diff
@@ -8787,7 +8787,12 @@ flatpak_dir_get_all_installed_refs (FlatpakDir  *self,
 
       for (i = 0; app_refs[i] != NULL; i++)
         {
-          FlatpakCollectionRef *ref = flatpak_collection_ref_new (NULL, app_refs[i]);
+          g_autofree char *remote = NULL;
+          g_autofree char *collection_id = NULL;
+          remote = flatpak_dir_get_origin (self, app_refs[i], NULL, NULL);
```

Comment:
Do you think it's okay to ignore errors here? My thinking was that otherwise this is more fragile than it was before.